docs/contributing/first_contribution_guide.md

```diff
@@ -156,7 +156,7 @@ Finally, before 'claiming' check you can do the following;
 
 ### 6. Contributing a solution
 
-**Important:** We only use GitHub's issue assignment feature to members of the Wagtail core team. If an issue is not assigned to anyone, feel free to work on it, **no need to ask "please assign me this issue"**.
+**Important:** If an issue is not assigned to anyone and doesn’t already have a pull request, feel free to work on it, **no need to ask "please assign me this issue"**. We only use GitHub's issue assignment feature to members of the Wagtail core team.
```

Comment:
This is very minor. Rephrase for clarity (state the general case first, exception after). Make it clear people should check whether there is already a PR or not.